runtime/ops/worker_host.rs

```diff
@@ -75,6 +76,34 @@ pub struct WorkerThread {
   message_closed: bool,
 }
 
+impl WorkerThread {
+  fn terminate(mut self) {
+    self.worker_handle.clone().terminate();
+    self
+      .join_handle
+      .take()
+      .unwrap()
+      .join()
+      .expect("Worker thread panicked")
+      .expect("Panic in worker event loop");
```

Comment:
I'm not sure if we should be `expect()`ing here. This would cause another panic if something goes wrong in worker and might be prone to other race conditions.

Reply:
Although that doesn't change current behavior so I guess it's fine for now